### DISCUSSION TITLE
How to login

### DISCUSSION THREAD

**Discussion by Ssnibles** [z-huang/music#513] [Q&A]
Hi whoever reads this
I was wondering on how to login, as I tried playing a song which was presumably age restricted? It didn't even start the song, it just stated that I have to Login to play this song.
Maybe I'm just blind, or it's a bug or simply not a feature yet, but I couldn't find anything anywhere.
Thanks to anyone that can answer this lol

**Comment by z-huang** [comment]
Login feature isn't available yet. See #88.

**Reply by Ssnibles** [reply]
Ok, thanks a bunch anyways, love the app so far :)
Might help out with a German translation when i have time
